### ISSUE TITLE
[1.19.2] Fabric Permissions API

### ISSUE THREAD

**Issue by itsdinkd** [coolsimulations/PocketDimensionPlots#2]
For 1.19.2, can you enable usage with Fabric permissions API? I believe its a great idea to allow permissions for this type of mod to use the pdp command, as you can set the required item to something like a barrier making it so only those with permissions can actually use their plot.

**Comment by coolsimulations** [comment]
Definitely, something I'll be doing in the next update. Only just noticed the permissions API existed after releasing the initial version.
